ui/src/components/pages/Namespace/index.tsx

```diff
@@ -157,18 +218,10 @@ export function Namespaces() {
         </Box>
       );
     }
-    if (!data) {
-      return (
-        <ErrorDisplay
-          title="Error loading namespace pipelines"
-          message="No resources found."
-        />
-      );
-    }
     return (
       <NamespacePipelineListing
         namespace={namespaceId || ""}
-        data={data}
+        data={data ? data : ({} as NamespaceSummaryData)}
```

Comment:
Please provide a proper NamespaceSummaryData object here when data is not available. Just set all values to 0 and empty arrays.

there is some data references in NamespacePipelineListing, want to avoid any undefined usage errors such as here: https://github.com/numaproj/numaflow/blob/f5ea9c1b8fcd22165e0a001c5dbf1ab9411d684f/ui/src/components/pages/Namespace/partials/NamespacePipelineListing/index.tsx#L98

Reply:
Done